.devcontainer/createCommand.sh

```diff
@@ -19,5 +19,11 @@ rm -f .env
 echo "PYTHONPATH=""$path"":$PYTHONPATH" >> .env
 echo "MYPYPATH=""$path"":$MYPYPATH" >> .env
 
+echo "Setting up git safe directory"
+git config --global --add safe.directory /workspaces/content
+
 echo "Setting up content dependencies"
-.hooks/bootstrap
\ No newline at end of file
+.hooks/bootstrap
+
+echo "Run demisto-sdk pre-commit to cache dependencies"
+poetry run demisto-sdk pre-commit >/dev/null 2>&1 || true
```

Comment:
why are you suppressing the pre-commit run result?

Reply:
We don't care about the result. It will just cache the pre-commit so `git commit` will be fast in the first run.